addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -992,4 +1006,16 @@ private void updateCubeState() {
     public boolean hasExcessDutyCycle() {
         return dutyCycle >= MAX_DUTY_CYCLE;
     }
+
+    public void backup() {
+        if (this.backup != 2) {
```

Comment:
This is if the backup is triggered by the user. In that case it gets a file name with the current date.
The 'automatic' one always has the same name (well, one per month to be precise), otherwise the system of the user may be flooded with backup messages from each restart of OH.
So, this indeed is intentional only if the backup is in progress